src/lib/chawathe98_matcher.rs

```diff
@@ -1285,144 +1244,149 @@ impl Config2 {
             if edge.edge_type == EdgeType::MOVE {
                 let edge_copy = edge.clone();
                 all_edges_move.insert((edge.from_node, edge.to_node, edge.edge_type));
                 let m_check = edge.from_node.is_leaf(&store.from_arena.borrow());
                 let n_check = edge.to_node.is_leaf(&store.to_arena.borrow());
-
                 if m_check && n_check {
-                    all_leaves_move.insert((edge_copy.from_node, edge_copy.to_node, edge_copy.edge_type));
+                    all_leaves_move.insert((edge_copy.from_node,
+                                           edge_copy.to_node,
+                                           edge_copy.edge_type));
                 }
             }
         }
         let mut all_leaves_parent =
-            HashSet::<(NodeId<FromNodeId>, NodeId<ToNodeId>, NodeId<FromNodeId>, NodeId<ToNodeId>)>::new();
+            HashSet::<(NodeId<FromNodeId>,
+                      NodeId<ToNodeId>,
+                      NodeId<FromNodeId>,
+                      NodeId<ToNodeId>)>::new();
         // Check if its nodes are in the leaf move operation
         for edge in all_leaves_move.clone() {
             let e1_parent = store.from_arena.borrow()[edge.0].parent().unwrap(); //edge.0.parent;
             let e1_siblings = e1_parent.children(&store.from_arena.borrow())
-                .collect::<Vec<NodeId<FromNodeId>>>();
+                                       .collect::<Vec<NodeId<FromNodeId>>>();
             let e1_to_node_parent = store.to_arena.borrow()[edge.1].parent().unwrap(); // edge to node parent
-
             let size_e1_siblings = e1_siblings.len();
             for _ in e1_siblings {
                 // are the siblings map to the same subtree with the parent being the same
                 let mut check_siblings_mapped_parent = 0;
                 // Check if siblings exists in move hashset
-                /*
-                loop through all the from node's siblings
-                    Main goal is to check if the move operation for its siblings exists
-                    1. loop though all the move leaves operations
-                        1.1) Checks
-                            1.1.1) if the parent the same as the current edge (From nodes)
-                            1.1.2) if the parent the same as the current edge (to nodes)
-                    Overall checking all the leaves and its parents the last two years
-                */
+                //loop through all the from node's siblings
+                //    Main goal is to check if the move operation for its siblings exists
+                //    1. loop though all the move leaves operations
+                //        1.1) Checks
+                //            1.1.1) if the parent the same as the current edge (From nodes)
+                //            1.1.2) if the parent the same as the current edge (to nodes)
+                //    Overall checking all the leaves and its parents the last two years
```

Comment:
The sentence beginning "overall" doesn't make sense. Rephrase.

Reply:
Fixed.